<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastramento_usuario</class>
 <widget class="QMainWindow" name="cadastramento_usuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro Usuario</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>LOGO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>276</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="cadastro_cargo_label_2">
       <property name="text">
        <string>Tipo Sanguíneo/RH:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_box2_sangues">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777213</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string> </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AB+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AB-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>O+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>O-</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="especialidade_label">
       <property name="text">
        <string>Especialidade:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="especialidade_line"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="especialidade_label_2">
       <property name="text">
        <string>Código CRMV:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="especialidade_line_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>277</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="cadastro_cargo_label_4">
       <property name="text">
        <string>Nível de Segurança:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_box2_sangues_2">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777213</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string> </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="cadastro_nome_label">
       <property name="text">
        <string>Nome Completo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_nome_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>183</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="cadastramento_idade_label">
       <property name="text">
        <string>Idade:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>205</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="cadastro_cargo_label">
       <property name="text">
        <string>Função:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_box_cargos">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777213</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentText">
        <string>--Selecione--</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string notr="true">--Selecione--</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Veterinário</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tratador</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="cadastro_cpf_label">
       <property name="text">
        <string>CPF:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_cpf"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>91</width>
      <height>153</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="foto_cadastramento">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>110</height>
        </size>
       </property>
       <property name="text">
        <string>FOTO 3X4</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_carregar_foto">
       <property name="text">
        <string>Foto</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
